benchmarks/build.gradle

```diff
@@ -39,12 +39,9 @@ dependencies {
             project(':grpc-interop-testing'),
             libraries.junit,
             libraries.mockito,
-            libraries.hdrhistogram
-    if (osdetector.os == "linux") {
-        // These are only available on linux.
-        compile libraries.netty_tcnative,
-                libraries.netty_transport_native_epoll
-    }
+            libraries.hdrhistogram,
+            libraries.netty_tcnative,
```

Comment:
Yes. Tcnative does provide OpenSSL for platforms other than Linux. Epoll is Linux-only, but not part of tcnative. I don't know if Netty is using the non-blocking I/O stuff from tcnative, but if it were that could also be available for other platforms.